lxdock/provisioners/base.py

```diff
@@ -91,6 +92,6 @@ def provision(self):
     # HELPER METHODS #
     ##################
 
-    def homedir_expanded_path(self, relative_path):
-        """ Expands the considered path with the absolute path of the home homedir. """
-        return os.path.join(self.homedir, relative_path)
+    def homedir_expanded_path(self, path):
+        """ Expands the considered path with the path of the home homedir if applicable. """
+        return os.path.join(self.homedir, path) if not os.path.isabs(path) else path
```

Comment:
In your initial commit, I wanted to suggest that you do that, but it turns out that it's not needed: https://docs.python.org/3/library/os.path.html#os.path.join:

`If a component is an absolute path, all previous components are thrown away and joining continues from the absolute path component.`

So I said nothing.